src/homework4/task7.py

```diff
@@ -0,0 +1,17 @@
+"""
+7. Оглянемся назад. Числа
+Даны два натуральных числа.
+Вычислите их наибольший общий делитель
+при помощи алгоритма Евклида
+(мы не знаем функции и рекурсию).
+"""
+
+a = 123
+b = 18
+
+while a and b:
+    if a > b:
+        a = a % b
+    else:
+        b = b % a
+print(a + b)
```

Comment:
It's a cheat - you just don't know where is result (second var is 0).
There is solution without this crutch